RELEASE_NOTES.md

```diff
@@ -0,0 +1,87 @@
+# Performance Optimizations
+## Intel Architecture Processors
+* Introduced initial support for future Intel Xeon processors with Intel AVX 10.2 and Intel AMX instruction sets support. 
+This functionality is not dispatched by default and requires opt-in with environment variable `ONEDNN_MAX_CPU_ISA=AVX10_2_512_AMX_2`.
+* Introduced initial support for future Intel Core processors with Intel AVX 10.2 instruction set support. This functionality is not dispatched by default and requires opt-in with environment variable `ONEDNN_MAX_CPU_ISA=AVX10_2_512`
+* Improved initialization time for convolution primitive when large number of threads used by introducing a new thread partition estimation and adjusting several blocking parameters. 
+* Improved performance of the following subgraphs with Graph API:
+    * [Scaled Dot Product Attention (SDPA)]  with implicit causal mask. 
+    * [Grouped Query Attention (GQA)] flavor specific for GEMMA models 
+
+[Scaled Dot Product Attention (SDPA)]: https://uxlfoundation.github.io/oneDNN/v3.9/dev_guide_graph_sdpa.html
+[Grouped Query Attention (GQA)]: https://uxlfoundation.github.io/oneDNN/v3.9/dev_guide_graph_gqa.html
+
+## Intel Graphics Products
+* Improved performance on Intel GPUs based on Xe3 architecture.
+* Improved matmul performance for Intel Arc Graphics for Intel Core Ultra processors (Series 2) (formerly Lunar Lake) 
+* Improved RNN primitive performance with LBR_GRU cell type  
+* Improved `int8` convolution performance with plain weights and trivial filter 
+* Improved `bf16` convolution with `NCHW` activations and plain weights on:
+    * Intel Arc Graphics for Intel Core Ultra processor series 2 (formerly Lunar Lake).
+    * Intel Arc B-series discrete graphics (formerly Battlemage).
+* Improved `fp32` softmax performance  
+* Improved performance of the following subgraphs with Graph API:
+    * SDPA  with implicit causal mask. 
+    * SDPA with bottom-right implicit causal mask. 
+    * fp32 SDPA. 
+    * fp16 SDPA on Intel GPUs without Intel XMX cores. 
+
+## AArch64-based Processors
+* Fixed failing layernorm test cases
+* Fixed out of bounds crash when ops are expected to be single threaded
+* Extended JIT depthwise convolution to `bf16`
+* Fixed failing binary add cases
+* Enable `f32` accumulation for `f16` matmul in ComputeLibrary
+* Fixed performance regression for 1x1 conv
+* Extended brdgemm to SVE 128
```

Comment:
This is not a user visible function. @Ryo-not-rio, can we reformulate this in terms of public functionality? I would assume the goal was improving depthwise convolution.